templates/actors/character-sheet.hbs

```diff
@@ -52,15 +52,16 @@
             {{!-- Header Attributes --}}
             <ul class="attributes flexrow">
                 <li class="attribute health">
-                    <h4 class="attribute-name box-title">{{ localize "DND5E.HitPoints" }}</h4>
+                    <h4 class="attribute-name box-title">{{localize "DND5E.HitPoints"}}</h4>
+                    <a class="config-button" data-action="hit-points"
+                        data-tooltip="{{localize 'DND5E.HitPointsConfig'}}"><i class="fas fa-cog"></i></a>
                     <div class="attribute-value multiple">
                         <input name="system.attributes.hp.value" type="text"
                             value="{{system.attributes.hp.value}}" placeholder="10"
-                            title="{{localize 'DND5E.HitPointsCurrent'}}" data-dtype="Number">
+                            data-tooltip="{{localize 'DND5E.HitPointsCurrent'}}" data-dtype="Number">
                         <span class="sep"> / </span>
-                        <input name="system.attributes.hp.max" type="text"
-                            value="{{system.attributes.hp.max}}" placeholder="10"
-                            title="{{localize 'DND5E.HitPointsMax'}}" data-dtype="Number">
+                        <div data-tooltip="{{#if source.attributes.hp.max}}{{localize 'DND5E.HitPointsOverride'}}
```

Comment:
Could we use a `<span>` for this instead of a `<div>`? that would be more consistent with how we handle non-editable labels elsewhere in the HTML structure.

Reply:
Hmm, I used `<div>` here specifically because I need to add some flex styling to match the spacing of the `<input>` without affecting the spacer in between. I could do this by specifically excluding the separator:

Current:
```css
&.multiple input, &.multiple div {
```

New:
```css
&.multiple input, &.multiple span:not(.sep) {
```